eval/mt_bench/components.py

```diff
@@ -0,0 +1,169 @@
+# type: ignore
+# pylint: disable=no-value-for-parameter,import-outside-toplevel,import-error
+from typing import List, NamedTuple
+from kfp.dsl import component, Input, Output, Artifact, Model, importer
+from utils.consts import PYTHON_IMAGE
+
+EVAL_IMAGE = "quay.io/sallyom/instructlab-ocp:eval"
+
+@component(base_image=EVAL_IMAGE, packages_to_install=["vllm"]))
```

Comment:
can we enforce a particular version?  `@git+https://github.com/opendatahub-io/vllm@2024.08.01`

Reply:
that was always my plan, but wasn't convenient while iterating on the 1000 different ways I tried to get vllm working properly - I'll add that to the Containerfile

Reply:
@tumido I'll update the Containerfile in a follow-up & remove the runtime install

Reply:
yes- will do that in a follow-up - will move the pip install to the Containerfile